### PR TITLE
Dev 29 - implemented html content for the email notifications

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [x] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #47

### CODE REVIEW

File: server/helpers/mailing.js [hunk 1]
```diff
@@ -1,21 +1,122 @@
...
+        color: black;
+        margin-top: 50px;"
+      >
+        I wish you a good day and thank you for using my project! 😇
+      </p>
+    </div>
+    <div style="padding: 20px;
+      margin-top: 50px;
+      background-color: #f2f2f2;
+      -webkit-box-shadow: 0px -5px 0px #bfbfbf;
```
Comment (NikolayYakovenko) [inline server/helpers/mailing.js]: As I know this property doesn't work in the most mail client.
But I am not sure.
You should know that not all css properties work in mail client.
https://templates.mailchimp.com/resources/email-client-css-support/

File: server/helpers/mailing.js [hunk 2]
```diff
@@ -1,21 +1,122 @@
...
+      display: block;
+      padding: 10px;
+      background: #00BCD4;"
+    >
+      Travel Stories
+    </div>
+    <div style="padding: 0 50px; max-width: 700px; margin: auto;">
+      <h2 style="text-align: center; color: black;">Password recovery</h2>
+
+      <p style="color: black;">Dear <b>${user.name} ${user.lastname}</b>!</p>
```
Comment (NikolayYakovenko) [inline server/helpers/mailing.js]: You can add `color: black` to parent **div** and avoid of code duplication.

Comment (KovDimaY) [reply server/helpers/mailing.js]: No, I cannot 😞 
It was my first approach but the "smart" behavior of the GMAIL adds classes to the content of the mail, making the same text as in the previous mails to be of different color (`#500050`). It made me crazy an I decided to add this style to all the elements with text so client could not change color when I do not want it... 😠 
Yes it is a bit dirty but at least it is secure `¯\_(ツ)_/¯`
If you know a way to solve my problem in a cleaner way - would be great to know! 😇

File: server/helpers/mailing.js (hunk 1 above)
Comment (KovDimaY) [reply server/helpers/mailing.js]: Yes, good point! Thank you! I will change it to border-top... 👍

File: server/helpers/mailing.js [hunk 3]
```diff
@@ -1,21 +1,122 @@
...
+      <p style="color: black;">
+        <b style="color: orange;">Warning!</b>
+        If you change your email, new notification will not arive to the current email anymore.
+      </p>
+      <p style="color: red;
+        font-size: 14px;
+        text-align: center;
+        font-weight: bold;"
+      >
+        If you get this emails but you were not changing you profile - your profile is in danger.
```
Comment (KovDimaY) [inline server/helpers/mailing.js]: "email", not "emails"

File: server/helpers/mailing.js [hunk 4]
```diff
@@ -1,21 +1,122 @@
...
+        <p style="color: black;">Last name: <b>${lastname}</b></p>
+        <p style="color: black;">Email/Login: <b>${email}</b></p>
+        ${newPassword ? `<p style="color: black;">Password: <b>${newPassword}</b></p>` : ''}
+      </div>
+      <p style="color: black;">
+        <b style="color: orange;">Warning!</b>
+        If you change your email, new notification will not arive to the current email anymore.
+      </p>
+      <p style="color: red;
+        font-size: 14px;
```
Comment (KovDimaY) [inline server/helpers/mailing.js]: I better do not write fixed font size as on mobile devices it works strangely. When I change the orientation of the screen - font size of the text where I did not put any syles is changing but the one with the fixed styles - no. And then it becomes to be smaller then I expect... 🤔

File: server/helpers/mailing.js (hunk 2 above)
Comment (NikolayYakovenko) [reply server/helpers/mailing.js]: Yes, I know about this behavior in gmail. But I had the same problem with link.
In this case your solution is ok.
Welcome to the world of email :smile:

**Pull request merged**